islandora_marcxml.module

```diff
@@ -97,3 +105,17 @@ function islandora_marcxml_islandora_ingest_steps_alter(array &$steps, array &$f
     }
   }
 }
+
+/**
+ * Implements hook_islandora_edit_datastream_registry().
+ */
+function islandora_marcxml_islandora_edit_datastream_registry(AbstractObject $object, AbstractDatastream $datastream) {
+  if ($datastream->id == 'MODS') {
+    return array(
+      'islandora_marcxml_update_mods_from_marcxml' => array(
+        'name' => t('Update MODS from MARCXML'),
```

Comment:
Would this exist, if it were stored? If it were stored, I would suspect it would be implemented as a derivative and you go to "regenerate" it.

Ideally I think we would also be able to provide something of a more verbose description, but that would require changing [how things get rendered](https://github.com/Islandora/islandora/blob/03b6afeb6f24fe92957821ad116367994e87da40/includes/datastream.inc#L320-L338) (not necessarily a bad idea as the existing implementation is rather simple); however, yeah... if we just want to be more verbose in the name here, that works too.